plugins/modules/system/java_keystore.py

```diff
@@ -233,30 +244,32 @@ def create_jks(module, name, openssl_bin, keytool_bin, keystore_path, password):
 
 
 def update_jks_perm(module, keystore_path):
-    file_args = module.load_file_common_arguments(module.params, path=keystore_path)
+    module.params['path'] = keystore_path
+    file_args = module.load_file_common_arguments(module.params)
```

Comment:
I had to modify this one as well, as one of the unit tests were failing with ansible-test.

Reply:
You need Ansible 2.10 / devel branch to use this module.

Reply:
Ok.

Reply:
What you could do is adjusting the module as follows: https://github.com/ansible-collections/community.crypto/commit/6b3ca2ca077000e656b18e3c64df1e72fd5174d0#diff-73b6f84151d5c073c62e4fb55b53113a Then it also works with 2.9.

Reply:
I've included such a change for java_keystore (and some more modules) in #306.

Reply:
Alright, thanks! I will resolve this one and fix the PEP issues, then squash and push my changes.

Reply:
#306 has now been merged, so you need to rebase to resolve the conflict.

Reply:
Done.